docs/HowToFAQ/FAQSingleton.md

```diff
@@ -0,0 +1,42 @@
+---
+title: How do I make a singleton instance of a class for repeated later use?
+---
+
+## Question
+
+How do I make a singleton instance of a class for repeated later use?
```

Comment:
Isn't the answer to this question that it can't be done?

Reply:
I think what the questioner was asking for can be done. If what you mean is that can it be that every attempt to allocate a class instance (`new C`) always results in the same instance (as opposed to a container delegating to a common instance) then I suppose it can't be done in Dafny.

Reply:
Maybe I'm misunderstanding the code below, but I don't think we can do a container delegating to a common instance either?

Reply:
Removed this FAQ for now, to be investigated further.